### ISSUE TITLE
Try model without 512->1280->512 transformation and pre-softmax tanh

### ISSUE THREAD

**Issue by mwinton** [mwinton/w266-final-project#27]
Yang's code is doing some things that don't make a lot of sense, but generate additional flops.

Specifically, in the attention layers they do a tanh immediately before softmax, which may not be good for gradient propagation during backprop.

Also, they use a dense layer to convert the 512-dim embeddings coming out to VGGNet to 1280-dim, but then convert back to 512-dim before adding to the question vector.  These also seems potentially problematic.  Since it's not described in their code, it may be a bug rather than a feature.
